dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -24,30 +39,32 @@ typedef struct _StaticShellEntry_
     HKEY hkClass;
 } StaticShellEntry, *PStaticShellEntry;
 
+#define DFM_FCIDM_SHVIEW_OFFSET 0x7000 // Offset from the menu ids in the menu resource to FCIDM_SHVIEW_*
```

Comment:
How do we know it has to be 0x7000 ?

Reply:
Because the code already had that magic value, I just gave it a sensible name.

shresdef.h does `#define IDM_PROPERTIES (FCIDM_SHVIEW_PROPERTIES - 0x7000)`